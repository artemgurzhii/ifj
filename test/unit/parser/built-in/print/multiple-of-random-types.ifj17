print 42; !"foo"; bar;
