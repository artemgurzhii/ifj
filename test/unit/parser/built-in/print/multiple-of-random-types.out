(print
  (int 42)
  (string 'foo')
  (id bar))
